### ISSUE TITLE
Command + Click (Control + Click on Windows, Control + Click on Ubuntu) on a link opens a new tab, switching to it.

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#6314]
**Describe the issue you encountered:**
Command + Click (Control + Click on Windows, Control + Click on Ubuntu) on a link opens a new tab, switching to it.

**Expected behavior:**
It should not, as the test plan says.

- Platform (Win7, 8, 10? macOS? Linux distro?): Windows 10 32bit

- Brave Version: 0.13.0 Preview5

- Steps to reproduce:
    1. Command + Click (Control + Click on Windows, Control + Click on Ubuntu) on a link
    2. It opens a new tab AND does switch to it.

- Any related issues:

**Comment by bsclifton** [comment]
I wasn't able to reproduce all of the time... but I was able to reproduce.

It happens on macOS also, with HEAD from brave/electron and HEAD from brave/browser-laptop.  

The steps I did:
- visit https://slashdot.org/
- hold down CMD and click lots of links
- One of the links I clicked (Zuckerberg, Jarvis) opened a new tab "about:blank" and switched focus to it.  It sat there for 1-2 seconds, then loaded the content
